src/pages/SearchPage.js

```diff
@@ -145,6 +142,7 @@ function SearchPage({betas, personalDetails, reports, isSearchingForReports}) {
 
     const onChangeText = (value = '') => {
         Report.searchInServer(searchValue);
+
```

Comment:
There is no reason to add this 🤔 
```suggestion
```

Reply:
whops, it probably got there when resolving conflicts. Thanks!